webui/react/src/services/types.ts

```diff
@@ -373,10 +373,22 @@ export interface AssignRolesToGroupParams {
   roleIds: number[];
 }
 
-export interface ListRolesParams {
+export interface AssignRolesToUserParams {
+  roleIds: number[];
+  userId: number;
+}
+
+export interface RemoveRoleFromUserParams {
+  roleId: number;
+  userId: number;
+}
+
+export interface ListRolesParams extends PaginationParams {
   isGlobal?: boolean;
-  limit?: number;
-  offset?: number;
+}
+
+export interface ListWorlspaceRolesParams extends PaginationParams {
```

Comment:
fix typo in `ListWorlspaceRolesParams` here and in api.ts / apiConfig.ts

Reply:
😅